deployment/appInsights.md

```diff
@@ -0,0 +1,20 @@
+## What is application insights?
```

Comment:
Please change casing of Application Insights to be consistent with the rest of the doc.

Please change casing of the MD file to be consistent with the other MDs in the repo. i.e. AppInsights.md